internal/controllers/sync-volume/utils.go

```diff
@@ -134,6 +134,9 @@ func getConverter(vol *stosv1.Volume, obj storageos.Object, log logr.Logger) fun
 			}
 		}
 
+		fmt.Printf("master hostname %s\n", stosVol.Master.Hostname)
```

Comment:
Do we need these lines in prod? If yes could you please convert it to logging?

Reply:
Not at all, those were using for debugging